index.html

```diff
@@ -27,11 +26,37 @@ <h1 id="" class="">Las paisas</h1>
 		</article>
 	</section>
 	<section>
-		<!-- comodities section -->
+    <!-- comodities section -->
+    <div class="carrouselContainer">
+      <h1 class="title-gallery">Galeria</h1>
+      <hr class="line">
+      <div class="description">
+        <span>
+          Nuestro hostal te ofrece todas las comodidades para que tengas la
+          mejor experiencia, observa todo lo que tenemos para ti y lo
+          majestuoso de CapurganĂ 
+        </span>
+      </div>
+
+      <div class="gallery">
+        <i class="fas fa-angle-left  back gallery__bottom"></i>
+        <img src="assets/img/IMG_8056.jpg" />
+        <i class="fas fa-angle-right ahead gallery__bottom"></i>
+      </div>
+      <!--<ol class="pointer">
+              <li  data-target="ahead" class="pointer__point " data-slide-to="0"></li>
+              <li  data-target="back" class="pointer__point " data-slide-to="1"></li>
+              <li  data-target="ahead" class="pointer__point " data-slide-to="2"></li>
+          </ol>-->
```

Comment:
please remove this if it is not used.

Reply:
that is the 3 points in zeplin, but i ain't how to make functional in js.
we were in trying to review it last Thursday

Reply:
ok, make them visible even if they don't work and lets schedule a call or meeting to get it working.